client/main.lua

```diff
@@ -185,7 +184,7 @@ function OpenMechanicActionsMenu()
 		elseif data.current.value == 'boss_actions' then
 			TriggerEvent('esx_society:openBossMenu', 'mechanic', function(data, menu)
 				menu.close()
-			end)
+			end), {wash = false}) -- set custom options, e.g disable washing
```

Comment:
Unexpected closing of function. Should be:

```
end, {wash = false}) -- set custom options, e.g disable washing
```
See: https://github.com/ESX-Org/esx_society#usage